core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -61,6 +61,30 @@
                    DEFAULT_NUM_COMMON_BLOCKING_TASK_THREADS,
                    value -> value > 0);
 
+    private static final long DEFAULT_DEFAULT_MAX_REQUEST_LENGTH = 10 * 1024 * 1024; // 10 MiB
+    private static final long DEFAULT_MAX_REQUEST_LENGTH =
+            getLong("defaultMaxRequestLength",
+                    DEFAULT_DEFAULT_MAX_REQUEST_LENGTH,
+                    value -> value >= 0);
+
+    private static final long DEFAULT_DEFAULT_MAX_RESPONSE_LENGTH = 10 * 1024 * 1024; // 10 MiB
+    private static final long DEFAULT_MAX_RESPONSE_LENGTH =
+            getLong("defaultMaxResponseLength",
+                    DEFAULT_DEFAULT_MAX_RESPONSE_LENGTH,
+                    value -> value >= 0);
+
+    private static final long DEFAULT_DEFAULT_REQUEST_TIMEOUT_MILLIS = 10 * 1000; // 10 seconds
+    private static final long DEFAULT_REQUEST_TIMEOUT_MILLIS =
+            getLong("defaultRequestTimeoutMillis",
+                    DEFAULT_DEFAULT_REQUEST_TIMEOUT_MILLIS,
+                    value -> value >= 0);
+
+    private static final long DEFAULT_DEFAULT_RESPONSE_TIMEOUT_MILLIS = 15 * 1000; // 15 seconds
```

Comment:
Could you add a small note about why it's longer than the default request timeout?